src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    @Override
+    public String toStorage() {
```

Comment:
I feel that you can consider renaming this method as 'toStorage' sounds ambiguous.